src/graph/entities/attribute_set.c

```diff
@@ -212,10 +212,14 @@ void AttributeSet_Set_Allow_Null
 		return;
 	}
 
-	size_t n = ATTRIBUTESET_BYTE_SIZE(_set) + sizeof(Attribute);
```

Comment:
How is this function (`AttributeSet_Set_Allow_Null`) different from `AttributeSet_Update`, meaning if we made a change here do we need to also apply a similar change to `AttributeSet_Update` ?

Reply:
not `AttributeSet_Update` `AttributeSet_Add` and now they are similar